crates/pathfinder/src/config.rs

```diff
@@ -35,6 +35,10 @@ pub enum ConfigOption {
     Integration,
     /// Chooses Testnet 2 network.
     Testnet2,
+    /// Specify the network.
+    Network,
+    /// Specify the StarkNet gateway URL.
+    Gateway,
```

Comment:
We may need two options here in the future for both `gateway` and `feeder_gateway`. This will require changing how we handle things in the Sequencer client, but maybe we want to do this in a follow-up PR?

Reply:
Is there any hint from Starkware that we have to be able to support different base URLs for these?

Reply:
Yes but only in the future. So either we do it now or later. I think I'll add some commits to do this tomorrow then.